### PR TITLE
Feature#30/crud branch

### LINKED ISSUE #30: CRUD for branches
# REQUIREMENTS
Implement the functional crud on the branches view for the adminsitrator

-  Use the view already made
- Use scaffold to implent the crud 

# DESIGN
image

### PR DESCRIPTION
**Pull request by nallely64**
# Description
 - What?: We've added the branch CRUD
 - Why?: To allow the admin to have control on the branches. 
 For more information see the issue #30 

  ## Screenshots
image

### CODE REVIEW

File: app/assets/stylesheets/pages/_administration_view.scss [hunk 1]
```diff
@@ -101,3 +101,15 @@
...
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  .test {
+    margin-left: 20px;
+    font-size: 20px;
+    width: 150px;
+  }
+}
```
Comment (YilKanda) [inline app/assets/stylesheets/pages/_administration_view.scss]: Add a blank line

File: app/controllers/branches_controller.rb [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        format.html { redirect_to branches_path, notice: "Branch was successfully updated." }
+      else
+        format.html { render :edit, status: :unprocessable_entity }
+      end
+    end
+  end
+
+  # DELETE /branches/1 or /branches/1.json
+  def destroy
+    @branch.destroy
```
Comment (YilKanda) [inline app/controllers/branches_controller.rb]: Try to make the delete logical.

File: app/helpers/branches_helper.rb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+module BranchesHelper
+end
```
Comment (YilKanda) [inline app/helpers/branches_helper.rb]: If you don't use it, you can delete it.

File: app/helpers/street_cps_helper.rb [hunk 4]
```diff
@@ -0,0 +1,2 @@
+module StreetCpsHelper
+end
```
Comment (YilKanda) [inline app/helpers/street_cps_helper.rb]: If you don't use it, you can delete it.

File: test/controllers/street_cps_controller_test.rb [hunk 5]
```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class StreetCpsControllerTest < ActionDispatch::IntegrationTest
+  # test "the truth" do
+  #   assert true
+  # end
```
Comment (YilKanda) [inline test/controllers/street_cps_controller_test.rb]: Code that you do not use, delete it.

File: app/assets/stylesheets/pages/_administration_view.scss (hunk 1 above)
Comment (juperameza) [reply app/assets/stylesheets/pages/_administration_view.scss]: Done

File: app/controllers/branches_controller.rb (hunk 2 above)
Comment (juperameza) [reply app/controllers/branches_controller.rb]: Done

File: app/helpers/street_cps_helper.rb (hunk 4 above)
Comment (juperameza) [reply app/helpers/street_cps_helper.rb]: DOne

File: app/helpers/branches_helper.rb (hunk 3 above)
Comment (juperameza) [reply app/helpers/branches_helper.rb]: Done

File: test/controllers/street_cps_controller_test.rb (hunk 5 above)
Comment (juperameza) [reply test/controllers/street_cps_controller_test.rb]: Done

**Pull request merged**